client/app/hosting/multisite/update/hosting-multisite-update.html

```diff
@@ -48,6 +48,25 @@
                     </div>
                 </div>
 
+                <div class="form-group" data-ng-if="hosting.isCloudWeb">
```

Comment:
Please refer to my previous comment on: `client/app/hosting/multisite/add/ovh/hosting-multisite-add-ovh.html`.